sale_stock_release_channel_partner_by_date/models/sale_order.py

```diff
@@ -42,7 +42,8 @@ def _compute_release_channel_id(self):
             if not rec._check_release_channel_partner_date_requirements():
                 continue
             channel_date = rec.release_channel_partner_date_id
-            rec.release_channel_id = channel_date.release_channel_id
+            if channel_date:
```

Comment:
```suggestion
            if not rec.release_channel_id and channel_date:
```
? Not sure, to check